### PR TITLE
chris-e-advice.html

### PR DESCRIPTION
**Pull request by Chriseden83**

### CODE REVIEW

**Comment by redrambles** [comment]
Hey @Chriseden83 !

This is a good start! We just need two more things and we'll be good to merge this in.

1) Please move your advice file into the advice directory. Right now it is inside the root of the project folder.

2) We still need a link to your advice page from the main index.html file, so that folks can find your words of wisdom, once your code is merged in. Students will typically add their link to the top or the bottom of that long list. :)

Once you move your advice file and add your link in the index.html file - add and commit your changes and then push your advice branch up once more, which will automatically update your pull request. Then we should be good to merge in your code!

Let us know if you have any questions!

Cheers,
Ann

**Comment by Chriseden83** [comment]
Added link and hopefully changed directory location but i'm not sure as it still reads skillcrush:master?

**Comment by redrambles** [comment]
Hey @Chriseden83 !

Thanks for making these changes! We're all good here except that we have one additional `advice-chris.css` file sitting in the root of the folder. Please delete that file.

Once you have done so, add that change with `git add -A` - commit as per usual and push up your advice branch once more. Then we should be all set to merge in your code.

Let us know if you have any questions!

Cheers,
Ann :)

**Comment by Chriseden83** [comment]
Hi Ann,

CSS file removed from root of advice. I had a few issues but hopefully this is correct?

Thanks
Chris

File: advice/chris-e-advice.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheet" type="text/css" href="./css/advice-chris.css">
```
Comment (redrambles) [inline advice/chris-e-advice.html:5]: We'll need this to be `href="../css/advice-chris.css"` (double dots at the beginning of the path)

File: index.html [hunk 2]
```diff
@@ -14,12 +14,16 @@ <h1>Skillcrush 105 - Advice for Newbies!</h1>
     <div class="all-advice-links">
       <div class="skillcrush">
 
+<<<<<<< HEAD
```
Comment (redrambles) [inline index.html:17]: Conflict marker to remove

File: index.html [hunk 3]
```diff
@@ -14,12 +14,16 @@ <h1>Skillcrush 105 - Advice for Newbies!</h1>
...
+                <a href="advice/chris-e-advice.html" target="_blank">Chris E.'s Advice</a><br>
+=======
```
Comment (redrambles) [inline index.html:19]: Conflict marker to remove

File: index.html [hunk 4]
```diff
@@ -14,12 +14,16 @@ <h1>Skillcrush 105 - Advice for Newbies!</h1>
...
                 <a href="advice/po-h-advice.html" target="_blank">Po H's Advice</a><br>
                 <a href="advice/luvia-m-advice.html" target="_blank">Luvia M's Advice</a><br>
                 <a href="advice/kathy-jj-advice.html" target="_blank">Kathy Jean's Advice</a><br>
                 <a href="advice/elizabeth-g-advice.html" target="_blank">Elizabeth G's Advice</a><br>
                 <a href="advice/davis-j-advice.html" target="_blank">Jordan D's Advice</a><br>
                 <a href="advice/molly-e-advice.html" target="_blank">Molly E's Advice</a><br>
+>>>>>>> upstream/master
```
Comment (redrambles) [inline index.html:26]: Conflict marker to remove

**Comment by redrambles** [comment]
Hey @Chriseden83 !

We are so close, I can taste it!!! Just a few last little changes to make. I've written you comments for each one - let me know if anything isn't clear. We're at the finish line!

Cheers,
Ann :)

**Comment by Chriseden83** [comment]
Hi Ann,

Thanks for the comments. I'm confident I've captured all the tweaks now.

I feel a lot more confident by troubleshooting what I've been doing wrong and it's actually helped me.

Chris

**Comment by acodeinprogress** [comment]
Hey @Chriseden83,

Great job! Your code is all [merged in](https://media.tenor.com/images/4a6158ad4f30c71f4fecf224cec12eb1/tenor.gif).


xoxo,
Sharon

**Pull request merged**